### DISCUSSION TITLE
Batterie, bzw. Akku deaktivieren falls nicht vorhanden

### DISCUSSION THREAD

**Discussion by spirit0607** [solectrus/solectrus#3531] [Q&A] [answered]
### Beschreibung (auf Deutsch oder Englisch)

Es wäre schön, wenn man die Anzeigen von Batterie, bzw. Akku komplett aus SOLECTRUS ausblenden, bzw. deaktivieren könnte. Falls man z.B SOLECTRUS mit einem Balkonkraftwerk oder kleiner PV Anlage betreibt, verfügt man in der Regel über keinen Stromspeicher. Die Anzeige zeigt dann logischerweise immer 0% Ladestand an, welches eher unschön ist.

**Comment by ledermann** [comment, marked as answer]
Danke für das Zuspielen dieses Balles, da kann ich direkt auf die [neue Doku](https://github.com/orgs/solectrus/discussions/3527) verweisen ;-)

> Wenn für einen Sensor keine Werte zur Verfügung stehen (weil man z.B. keine Wärmepumpe hat), muss der Sensor “leer” definiert werden, und zwar so:
>
> INFLUX_SENSOR_HEATPUMP_POWER=

https://docs.solectrus.de/referenz/dashboard/konfiguration#umgebungsvariablen-f%C3%BCr-die-sensor-konfiguration-ab-version-015

In deinem Fall muss also nur Folgendes in der `.env` stehen:

```env
INFLUX_SENSOR_BATTERY_SOC=
INFLUX_SENSOR_BATTERY_CHARGING_POWER=
INFLUX_SENSOR_BATTERY_DISCHARGING_POWER=
```

Außerdem darauf achten, dass diese Variablen auch unter `services.dashboard.environment` aufgelistet werden:
https://docs.solectrus.de/support/docker#was-ist-eine-env-datei

Der Akku verschwindet dann komplett aus dem UI.

**Comment by spirit0607** [comment]
Ach super, das wusste ich tatsächlich nicht 👍🏼 Klasse!
